resources/views/layouts/base-h2.html.twig

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="{{ appLocale }}">
+  {# Note that the </head> and <body> tags are located inside the common/head
+  file due to Google Tag Manager being required on all pages. #}
+  <head>
+    <title>{% block title %}{% endblock %}</title>
+    {% block socialMeta %}{% endblock %}
+    {% include "common/head" with {"hydrogen": true} %}
+  {# </head> #}
+  {# <body> #}
+  <div data-h2-system>
+    <main>
+      {% block body %}{% endblock %}
+    </main>
+  </div>
+  {% include "common/footer" %}
+  {% include "common/scripts" %}
+  {% block scripts %}{% endblock %}
+  <script async defer
+    src="https://cdn.jsdelivr.net/npm/@hydrogen-design-system/system@latest/dist/compiled/latest/system.min.js">
+  </script>
```

Comment:
This is already in the included scripts template, it doesn't need to be here as well.